interface/i18n/mist.it.i18n.json

```diff
@@ -110,34 +139,47 @@
             "runningNodeFound": "Trovato nodo Ethereum in esecuzione!",
             "nodeConnectionTimeout": "Impossibile avviare un nodo Ethereum!<br><small>Se <a href='https://www.ethereum.org/cli#geth' class='button' target='_blank'>hai installato Geth</a>, usa questo comando per eseguirlo:<br> <code>geth --ipcpath __path__</code></small><br> <small><a href='https://github.com/ethereum/mist/issues' class='button' target='_blank'> o riporta il problema </a></small>",
             "nodeBinaryNotFound": "Impossibile trovare l'eseguibile del nodo Ethereum !<br><small> <a href='https://www.ethereum.org/cli#geth' class='button' target='_blank'> Per favore avvialo manualmente prima. </a> </small>",
-            "nodeSyncing": "Il nodo Ethereum deve sincronizzarsi, attendere prego...",
-            "nodeSyncInfo": "Scaricamento blocco __displayBlock__ di __highestBlock__.",
-            "nodeSyncConnecting": "Alla ricerca di nodi...",
             "nodeStarting": "Nodo Ethereum in avvio...",
             "nodeStarted": "Nodo Ethereum avviato",
             "nodeConnected": "Nodo Ethereum connesso",
             "nodeStopping": "Nodo Ethereum in arresto...",
             "nodeStopped": "Nodo Ethereum arrestato",
             "nodeError": "Nodo Ethereum errore di connessione :'(",
             "unableToBindPort": "Nodo Ethereum non in esecuzione. Esiste altra istanza in esecuzione?",
-            "nodeSyncInfoStates": "Scaricamento blocco __displayBlock__ of __highestBlock__, <br> Scaricamento struttura chain __displayState__ of __displayKnownStates__",
-            "nodeSyncFoundPeers": "Connessione a __peers__ peers...",
+            "nodeSyncing": "Il nodo Ethereum deve sincronizzarsi, attendere prego...",
+            "nodeSyncInfo": "Scaricamento blocco __displayBlock__ di __highestBlock__.",
+            "nodeSyncConnecting": "Alla ricerca di nodi...",
+            "nodeSyncInfoStates": "Scaricamento blocco __displayBlock__ di __highestBlock__, <br> Scaricamento struttura chain __displayState__ di __displayKnownStates__",
+            "nodeSyncFoundPeers": "Connessione a __peers__ nodi ...",
             "launchApp": "Avvia Applicazione",
+            "retryConnection": "Riprova la connessione",
             "clientBinaries": {
                 "scanning": "Controlla aggiornamento nodo...",
                 "downloading": "Scaricamento nuovo nodo...",
-                "loadConfig": "Caricamento configurazione client...",
-                "filtering": "Filtro codice binario del client...",
-                "done": "Nodo Ethereum  aggiornato...",
+                "loadConfig": "Caricamento configurazione cliente...",
```

Comment:
is better to use "client" untranslated, cliente means customer and here it's wrong